testdata/well_tested/src/numbers.rs

```diff
@@ -6,6 +6,14 @@ fn is_double(a: u32, b: u32) -> bool {
     a == 2 * b
 }
 
+fn negate_int(a: i32) -> i32 {
```

Comment:
I think we should also test bools in some other file alongside this.

Reply:
I added the file `testdata/well_tested/src/booleans.rs`